lib/pry/commands/edit.rb

```diff
@@ -157,7 +157,7 @@ def input_expression
       case opts[:i]
       when Range
         (_pry_.input_array[opts[:i]] || []).join
-      when Fixnum
+      when 1.class
```

Comment:
This is a cool idea, but we could probably just make it `when Integer` since Integer still exists as a superclass of Fixnum/Bignum in earlier versions of Ruby.